pkg/image/importer/importer.go

```diff
@@ -509,6 +510,14 @@ func (isi *ImageStreamImporter) importRepositoryFromDocker(ctx gocontext.Context
 				continue
 			}
 		}
+
+		signatures, err := signature.GetSignatureForImage(repository.Registry.Hostname(), repository.Name, importDigest.Image.Name)
+		if err == nil {
+			importDigest.Image.Signatures = signatures
```

Comment:
@mtrmac good catch, but I think this code only runs when we creating images.

IOW. we don't update the images after we create them or track any changes to an image itself once imported.
We do track changes to imagestreams which points to an docker repository. When the SHA256 change (iow. new image is pushed), we run the import to pull the updated images in.

Also in this case, in order to verify the rhel7:latest image imported in "openshift" namespaces (which results into `registry/openshift/rhel7:latest` you have to use --expected-identity to verify).

@soltysh FYI